kivy/uix/relativelayout.py

```diff
@@ -290,6 +290,15 @@ def _apply_transform(self, m, pos=None):
         m.translate(self.x, self.y, 0)
         return super(RelativeLayout, self)._apply_transform(m, (0, 0))
 
+    def on_motion(self, etype, me):
+        if me.type_id in self.motion_filter and 'pos' in me.profile:
+            me.push()
+            me.apply_transform_2d(self.to_local)
```

Comment:
probably a change for later, but a context manager version of this function would probably be nice and allow simplifying such code to:
```py
with me.transform_2d(self.to_local):
    return super().on_motion(etype, me)
```

something like
```py
from contextlib import contextmanager

@contextmanager
def transform_2d(self, transform):
    self.push()
    self.apply_transform_2d(transform)
    try:
        yield
    finally:
        self.pop()
```

Reply:
That can be added in another pull request, a feature for `MotionEvent` class.